src/test/java/core/basesyntax/HelloWorldTest.java

```
@@ -1,8 +1,118 @@
+
+        registerService.register(user);
+        User newUser = storageDao.get(user.getLogin());
+
+        assertEquals(newUser.getLogin(), user.getLogin());
+    }
+
+    @Test
+    void register_nullAge_NotOk() {
+        user.setLogin("Igor123");
+        user.setPassword("123456");
+        user.setAge(null);
+
+        assertThrows(NullPointerException.class, () -> registerService.register(user));
+    }
+
+    @Test
+    void register_nullLogin_NotOk() {
+        user.setLogin(null);
+        user.setPassword("123456");
+        user.setAge(19);
+
+        assertThrows(NullPointerException.class, () -> registerService.register(user));
+    }
+
+    @Test
+    void register_nullPassword_NotOk() {
+        user.setLogin("Igor123");
+        user.setPassword(null);
+        user.setAge(19);
+
+        assertThrows(NullPointerException.class, () -> registerService.register(user));
+    }
+
+    @Test
+    void register_sizePassword_notOk() {
+        user.setLogin("Igor123");
+        user.setPassword("1234");
+        user.setAge(19);
+
+        User actual = registerService.register(user);
+        assertNotEquals(actual, user);
+    }
+
+    @Test
+    void register_sizeLogin_notOk() {
+        user.setLogin("Igor");
+        user.setPassword("123456");
+        user.setAge(19);
+
+        User actual = registerService.register(user);
+        assertNotEquals(actual, user);
+    }
+
+    @Test
+    void register_sizeAge_notOk() {
+        user.setLogin("Igor123");
+        user.setPassword("123456");
+        user.setAge(14);
+
+        User actual = registerService.register(user);
+        assertNotEquals(actual, user);
+    }
+
+    @Test
+    void register_duplicateUser_NotOk() {
+        user.setLogin("Igor123");
+        user.setPassword("123456");
+        user.setAge(19);
+
+        User actual = registerService.register(user);
+        User expected = registerService.register(user);
+
+        assertNotEquals(actual, expected);
```

Issue: For duplicate users, the test checks that the returned user is not equal to the expected user. According to the requirements, the method should throw an exception (such as `IllegalArgumentException`) when a user with the same login already exists, not return a user object. Please update this test to expect an exception.